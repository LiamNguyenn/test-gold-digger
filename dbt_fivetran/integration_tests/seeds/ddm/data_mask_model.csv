user_id,email,phone_number
1,alex@testing.com,1234567890
1,tu@testing.com,1234567890
