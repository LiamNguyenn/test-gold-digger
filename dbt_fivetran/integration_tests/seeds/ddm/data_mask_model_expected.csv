user_id,email,phone_number
1,970a74137fcc9b20e4da9ce94fc2cf0b,e807f1fcf82d132f9bb018ca6738a19f
1,a57f07ac0603e43defb61327f2174454,e807f1fcf82d132f9bb018ca6738a19f
